GitHubProjectColumns.ps1

```diff
@@ -0,0 +1,381 @@
+        Id of the column to modify.
+
+    .PARAMETER Name
+        The name for the column.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no command line status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Set-GitHubProjectColumn -Column 999999 -Name NewColumnName
+
+        Set the project column name to 'NewColumnName' with column with id 999999.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess)]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification = "Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
+    param(
+        [Parameter(Mandatory)]
+        [int64] $Column,
+
+        [Parameter(Mandatory)]
+        [string] $Name,
+
+        [string] $AccessToken,
+
+        [switch] $NoStatus
+    )
+
+    Write-InvocationLog
+
+    $telemetryProperties = @{}
+
+    $uriFragment = "/projects/columns/$Column"
+    $apiDescription = "Updating column $Column"
+
+    $hashBody = @{
+        'name' = $Name
+    }
+
+    $params = @{
+        'UriFragment' = $uriFragment
+        'Description' = $apiDescription
+        'Body' = (ConvertTo-Json -InputObject $hashBody)
+        'AccessToken' = $AccessToken
+        'Method' = 'Patch'
+        'TelemetryEventName' = $MyInvocation.MyCommand.Name
+        'TelemetryProperties' = $telemetryProperties
+        'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+        'AcceptHeader' = 'application/vnd.github.inertia-preview+json'
+    }
+
+    return Invoke-GHRestMethod @params
+}
+
+function Remove-GitHubProjectColumn
+{
+<#
+    .DESCRIPTION
+        Removes the column for a project.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER Column
+        Id of the column to remove.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no command line status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
```

Comment:
Can you include an example that shows using `-Confirm:$false` [like was done here](https://github.com/microsoft/PowerShellForGitHub/blob/3fab72464e38cb573408add7e99d5a6bb0db2ea1/GitHubRepositories.ps1#L504)?